### ISSUE TITLE
BlenderBim - custom made doors show mesh representation in plan svg in addition to 2d

### ISSUE THREAD

**Issue by Plae-Blue-Dot** [IfcOpenShell/IfcOpenShell#2750]
Seems to be the case since blenderbim-230203 and up +

Model:
![image](https://user-images.githubusercontent.com/78321676/217531066-452c74f1-bfcb-43eb-aecf-a320340dd978.png)

SVG working fine with blenderbim-230202 and older:
![image](https://user-images.githubusercontent.com/78321676/217531251-1cb6300a-fbeb-42be-a350-88f8a73c93cc.png)

SVG right side custom door broken with blenderbim-230203 and newer:
![image](https://user-images.githubusercontent.com/78321676/217531219-b71daec3-bf40-468f-8f62-305244072acc.png)

Something to do with the 2d representation changes?
How could I update these to work as expected?

**Comment by Moult** [comment]
This is part of an intentional change here: https://github.com/IfcOpenShell/IfcOpenShell/commit/6adbcdd8e1dae7ba8dceee2d6dcfd365082e1c43

In short, the previous behaviour is that "Annotation" representations would replace the "Body" representations if they were found and relevant to the target view (e.g. PLAN_VIEW) of the drawing.

The new behaviour is that the "Annotation" representation is in addition to, instead of replacing the Body representation. This is a bit of an ambiguous area of IFC and there are no strict rules on whether this is correct or wrong, so we're doing a bit of trailblazing here. So if you want to replace the 3D body representation with a 2D one, your door swing should now be in the Plan/Body/PLAN_VIEW context.

This finally fixes the issue of how to have things like windows and joinery which have elevation annotation (e.g. hinge swing triangles, door sliding arrows) that are generally drawn as 3D curves (i.e. Model/Annotation/ELEVATION_VIEW) superimposed on top of a 3D solid body (i.e. Model/Body/MODEL_VIEW).

TL;DR edit your Plan/Annotation/PLAN_VIEW to be Plan/Body/PLAN_VIEW for a quick fix.

Also relevant bug here https://github.com/IfcOpenShell/IfcOpenShell/issues/2744 which I'm curious what @aothms says.
